backend/entities/office_hours/oh_event_entity.py

```diff
@@ -50,10 +52,72 @@ class OfficeHoursEventEntity(EntityBase):
     )
 
     # NOTE: Unidirectional relationship to Room
-    location_id: Mapped[int] = mapped_column(ForeignKey("room.id"), nullable=False)
-    location: Mapped["RoomEntity"] = relationship("RoomEntity")
+    room_id: Mapped[int] = mapped_column(ForeignKey("room.id"), nullable=False)
+    room: Mapped["RoomEntity"] = relationship("RoomEntity")
 
     # NOTE: One-to-many relationship of OfficeHoursEvent to tickets
     tickets: Mapped[list["OfficeHoursTicketEntity"]] = relationship(
-        back_populates="event", cascade="all, delete"
-    )
+        back_populates="event", cascade="all, delete")
```

Comment:
One quick, small thing! Since we just changed the name of the event field in `OfficeHoursTicketEntity` from `event` to `oh_event`, we probably need to change this back populates reference to `oh_event`! 

Looks great otherwise!!

Reply:
This has been updated now, thanks!!